app/controllers/api/v1/concerns/api_caching.rb

```diff
@@ -4,13 +4,6 @@ module Concerns
       module ApiCaching
         extend ActiveSupport::Concern
 
```

Comment:
Were these just lingering no-longer used parameters?  (I'd check myself, but I don't have a coding laptop right now... grrrr...)

Reply:
Yes - it all got moved to `RequestUtils` a while back.